cmd/resetHeaderCheckCount.go

```diff
@@ -65,6 +65,13 @@ func init() {
 func resetHeaderCount(blockNumber int64) error {
 	blockChain := getBlockChain()
 	db := utils.LoadPostgres(databaseConfig, blockChain.Node())
+
+	genConfig, configErr := prepConfig()
+	if configErr != nil {
+		LogWithCommand.Fatalf("SubCommand %v: failed to prepare config: %v", SubCommand, configErr)
+		return configErr
+	}
+
 	repo, repoErr := repositories.NewCheckedHeadersRepository(&db, genConfig.Schema)
```

Comment:
Seeing this makes me wonder if it'd be beneficial to have a helper for loading a single value from the config rather than loading the whole thing, though that definitely doesn't need to be a part of this PR

Reply:
yeah, that is a good idea. i've added a new story: https://makerdao.atlassian.net/browse/VDB-1730